pkg/apis/hive/v1alpha1/hiveconfig_types.go

```diff
@@ -25,6 +25,10 @@ type HiveConfigSpec struct {
 	// CA generated by each cluster on installation.
 	// +optional
 	AdditionalCertificateAuthorities []corev1.LocalObjectReference `json:"additionalCertificateAuthorities,omitempty"`
+
+	// GlobalPullSecret should be used when additional secret needs to be given apart from PullSecret.
```

Comment:
Should add to comments that the registries will be merged with any cluster specific pull secrets, and the secret is assumed to be in the "hive" namespace.

Reply:
Will fix it.